samwise.js

```diff
@@ -5,37 +5,36 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.